### DISCUSSION TITLE
REEMBOLSO LICENÇA

### DISCUSSION THREAD

**Discussion by wagner521** [aclap-dev/video-downloadhelper#1047] [License & Purchase requests]
### What's your issue?

SOLICITO O REEMBOLSO DA MINHA COMPRA NO CHROME JA QUE NÃO POSSO BAIXAR VIDEOS DO YOUTUBE. COM O REEMBOLSO FAREI NOVAMENTE A COMPRA NO FIREFOX .

### The 10 first digits (only 10) of your license number or email address

2d00936c4f

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by julianosoares1990** [comment]
I request a refund for the license key 21095320..., as I cannot download the complete videos, so it is no longer of any use to me.

**Comment by jcfrog** [comment]
@julianosoares1990 Hi,
You have been refunded. Should be available in a few business days.
Please do not share license keys on public forums.

**Comment by vander0131** [comment]
Solicito o reembolso da chave de licença id 45731aae5b...  foi feito uma compra acidental. pois minhas informaçoes bancarias e do cartao estava cadastrado no google pay.

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Reply by vander0131** [reply]
obrigado pela solução.

**Comment by PedroFernandes24** [comment]
Hello. I request a refund for the license key id f5f9d86ab4 ... because it did not download the videos correctly. Thank you

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by valeria9081** [comment]
Qual é o seu problema?
SOLICITO O REEMBOLSO DA MINHA COMPRA POIS NÃO CONSEGUIR ABAIXAR OS VIDEOS QUE EU QUERIA.

Os 10 primeiros dígitos (apenas 10) do seu número de licença ou endereço de e-mail
917340ced

Confirmo que não colei meu número de licença completo

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by alesalles31** [comment]
Hello. I request a refund for the license key id 519dbb6f4d ... because it did not download the videos correctly. Thank you

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by erikcardoso777-lang** [comment]
Olá. Solicito um reembolso para o ID da chave de licença 200011235 porque não baixou os vídeos corretamente. Obrigado

**Reply by jcfrog** [reply]
Hi
Please send an email to licenses@downloadhelper.net
